ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template3.kt

```diff
@@ -0,0 +1,199 @@
+        ) {
+            Template3MainContent(state, viewModel)
+        }
+        PurchaseButton(state, viewModel)
+        Footer(templateConfiguration = state.templateConfiguration, viewModel = viewModel)
+    }
+}
+
+@Composable
+private fun Template3MainContent(state: PaywallViewState.Loaded) {
+    IconImage(state.templateConfiguration.images.iconUri)
+    val localizedConfig = state.selectedLocalization
+    val colors = state.templateConfiguration.getCurrentColors()
+    Text(
+        style = MaterialTheme.typography.headlineSmall,
+        fontWeight = FontWeight.Black,
+        textAlign = TextAlign.Center,
+        text = localizedConfig.title,
+        color = colors.text1,
+    )
+    Spacer(modifier = Modifier.height(UIConstant.defaultVerticalSpacing * 2))
+    Features(
+        features = localizedConfig.features,
+        colors = colors,
+    )
+}
+
+@Composable
+private fun Features(
+    features: List<PaywallData.LocalizedConfiguration.Feature>,
+    colors: TemplateConfiguration.Colors,
+) {
+    if (features.isEmpty()) return
+
+    Column(
+        modifier = Modifier.verticalScroll(rememberScrollState())
+            .fillMaxHeight(),
+        verticalArrangement = Arrangement.spacedBy(UIConstant.defaultVerticalSpacing, Alignment.Top),
+    ) {
+        features.forEach { feature ->
+            Feature(
+                feature = feature,
+                colors = colors,
+            )
+        }
+    }
+}
+
+@Composable
+private fun Feature(
+    feature: PaywallData.LocalizedConfiguration.Feature,
+    colors: TemplateConfiguration.Colors,
+) {
+    Row(
+        verticalAlignment = Alignment.Top,
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(horizontal = UIConstant.defaultHorizontalPadding),
+    ) {
+        feature.iconID?.let { PaywallIconName.fromValue(it) }?.let { icon ->
+            Box(
+                modifier = Modifier
+                    .size(Template3UIConstants.featureIconSize)
+                    .clip(CircleShape)
+                    .background(colors.accent2),
+            ) {
+                PaywallIcon(
+                    icon = icon,
+                    tintColor = colors.accent1,
+                    modifier = Modifier.padding(Template3UIConstants.iconPadding),
+                )
+            }
+        }
+        Column(
+            modifier = Modifier
+                .padding(start = UIConstant.defaultHorizontalPadding),
+        ) {
+            Text(
+                style = MaterialTheme.typography.bodyLarge,
+                fontWeight = FontWeight.Bold,
+                textAlign = TextAlign.Start,
+                text = feature.title,
+                color = colors.text1,
+            )
+            feature.content?.let { content ->
+                Text(
+                    modifier = Modifier.padding(top = Template3UIConstants.iconPadding * 2),
```

Comment:
Slightly strange to use the icon padding for the text... But it's probably ok. We can polish all this later.